mkdocs.yml

```diff
@@ -10,6 +10,7 @@ nav:
     - Quickstart: dataset/quickstart.md
     - Installing <code>kolena</code>: installing-kolena.md
     - Core Concepts: dataset/core-concepts/index.md
+    - Use Cases: dataset/use-cases.md
```

Comment:
Was thinking if it'd be helpful to include a ref at the bottom of the quick start page to reference this page, maybe in the conclusion section. Any thoughts?

Reply:
Oh I was thinking just a sentence in the quick start guide, such as check out other examples in the use cases section. Agreed that we don't want to include this whole section in the quick start guide.

Reply:
Sorry I misunderstood. Okay I updated to include that. I think its a great idea to have that.